axelrod/strategies/__init__.py

```diff
@@ -7,17 +7,36 @@
 # because it creates circular dependencies
 
 from .meta import (
-    MetaPlayer, MetaMajority, MetaMinority, MetaWinner, MetaHunter,
-    MetaMajorityMemoryOne, MetaWinnerMemoryOne, MetaMajorityFiniteMemory,
-    MetaWinnerFiniteMemory, MetaMajorityLongMemory, MetaWinnerLongMemory,
-    MetaMixer, MetaWinnerEnsemble
+    MetaHunter, MetaHunterAggressive, MetaPlayer, MetaMajority,
+    MetaMajorityMemoryOne, MetaMajorityFiniteMemory, MetaMajorityLongMemory,
+    MetaMinority, MetaMixer, MetaWinner, MetaWinnerDeterministic,
+    MetaWinnerEnsemble, MetaWinnerMemoryOne, MetaWinnerFiniteMemory,
+    MetaWinnerLongMemory, MetaWinnerStochastic, MWEDeterministic,
+    MWEFiniteMemory, MWELongMemory, MWEMemoryOne, MWERandom, MWEStochastic
     )
 
-all_strategies.extend([MetaHunter, MetaMajority, MetaMinority, MetaWinner,
-                       MetaMajorityMemoryOne, MetaWinnerMemoryOne,
-                       MetaMajorityFiniteMemory, MetaWinnerFiniteMemory,
-                       MetaMajorityLongMemory, MetaWinnerLongMemory, MetaMixer,
-                       MetaWinnerEnsemble])
+all_strategies += [MetaHunter,
```

Comment:
Is this just a stylistic change?

(I don't mind at all, not requesting a change here, just curious.)